src/app/modules/submission/email-submission/email-submission.middleware.ts

```diff
@@ -71,7 +76,7 @@ export const receiveEmailSubmission: RequestHandler<
     })
     .map((parsed) => {
       req.body = parsed
-      const hashes = EmailSubmissionService.hashSubmission(parsed)
+      const hashes = EmailSubmissionService.hashResponses(parsed)
```

Comment:
in your opinion, do you think we have sufficient tests to make sure that `hashResponses` is called with `parsed` the same way every time?

Reply:
I don't think we currently have any tests which check that the service functions are called correctly. But in general we're not spending the effort to write tests for middleware functions which are going to be collapsed into controller functions anyway. When that happens, we should definitely write tests to ensure that all the services are being called correctly.